MachObfuscator/Options/Options.swift

```diff
@@ -1,16 +1,16 @@
 import Foundation
 
 struct Options {
-    var help: Bool
-    var quiet: Bool
-    var verbose: Bool
-    var debug: Bool
-    var methTypeObfuscation: Bool
-    var machOViewDoom: Bool
-    var swiftReflectionObfuscation: Bool
-    var obfuscableFilesFilter: ObfuscableFilesFilter
-    var manglerType: SymbolManglers?
-    var skipSymbolsFromSources: [URL]
+    var help = false
```

Comment:
I think this will make the compilation process a bit slower, but the project is probably to small to feel that. Anyway I love placing default values here <3